web-ui/src/main/resources/catalog/components/metadataactions/MetadataActionsDirective.js

```diff
@@ -728,6 +728,21 @@
           scope.userGroupDefined = false;
           scope.userGroups = null;
 
+          scope.ownerUserName = "";
+          scope.ownerGroupName = "";
+
+          if (ownerId) {
+            $http.get("../api/users/" + ownerId).then(function (response) {
```

Comment:
Maybe not an issue, but to get user info, current user need to be admin or userAdmin or the user. Not sure if transfer ownership is displayed for other type of users eg. reviewer ?

Reply:
This option is only available to administrators, it should be fine.